templates/casper-node.yml

```diff
@@ -195,9 +195,9 @@ Resources:
                 curl -JLO https://bintray.com/casperlabs/debian/download_file?file_path=casper-client_0.9.3-0_amd64.deb
                 curl -JLO https://bintray.com/casperlabs/debian/download_file?file_path=casper-node-launcher_0.3.1-0_amd64.deb
                 apt install -y ./casper-client_0.9.3-0_amd64.deb ./casper-node-launcher_0.3.1-0_amd64.deb
-                sudo -u casper /etc/casper/pull_casper_node_version.sh 1_0_0 ${NetworkName}
+                sudo -u casper /etc/casper/pull_casper_node_version.sh 1_0_0 ${CasperNetwork}
                 sudo -u casper /etc/casper/config_from_example.sh 1_0_0
-                sed -i "/trusted_hash =/c\trusted_hash = '$(curl -s ${NetworkIp}:8888/status | jq -r .last_added_block_info.hash | tr -d '\n')'" /etc/casper/1_0_0/config.toml
+                sed -i "/trusted_hash =/c\trusted_hash = '$(curl -s ${TrustedHashIp}:8888/status | jq -r .last_added_block_info.hash | tr -d '\n')'" /etc/casper/1_0_0/config.toml
```

Comment:
Lets simplify:

sed -i "/trusted_hash =/c\trusted_hash = '$(curl -s ${TrustedHashIp}:8888/status | jq -r .last_added_block_info.hash | tr -d '\n')'" /etc/casper/1_0_0/config.toml

to 

sed -i "/trusted_hash =/c\trusted_hash = ${TrustedHash}'" /etc/casper/1_0_0/config.toml

Reply:
Actually the file you commented is outdated. I changed the script to use this one https://github.com/make-software/how-to-casperlabs/blob/master/docs/aws/setup-testnet-validator-from-scratch.md#get-known-validator-ip Do you still want us to create the TrustedHash parameter?

Reply:
Or should we remove it?